packages/victory-area/package.json

```
@@ -25,7 +25,7 @@
     "victory-vendor": "^36.4.0"
   },
   "peerDependencies": {
-    "react": "^16.6.0 || ^17.0.0"
+    "react": "^16.6.0 || ^17.0.0 || ^18.0.0"
```

Maybe we should change this to;
```suggestion
    "react": ">=16.8.0"
```
For people who are early-adopters of React, this is helpful.

---

👍